docs/hosting/server-setups/digital-ocean.md

```diff
@@ -0,0 +1,178 @@
+DOMAIN_NAME=example.com
+
+# The subdomain to serve from
+SUBDOMAIN=n8n
+
+# DOMAIN_NAME and SUBDOMAIN combined decide where n8n will be reachable from
+# above example would result in: https://n8n.example.com
+
+# The user name to use for authentication - IMPORTANT ALWAYS CHANGE!
+N8N_BASIC_AUTH_USER=user
+
+# The password to use for authentication - IMPORTANT ALWAYS CHANGE!
+N8N_BASIC_AUTH_PASSWORD=password
+
+# Optional timezone to set which gets used by Cron-Node by default
+# If not set New York time will be used
+GENERIC_TIMEZONE=Europe/Berlin
+
+# The email address to use for the SSL certificate creation
+SSL_EMAIL=example@example.com
+```
+
+## Create Docker Compose file
+
+The Docker compose file defines the services the application needs, in this case Caddy and n8n.
+
+- The Caddy service definition defines the ports it uses and the local volumes to copy to the containers.
+- The n8n service definition defines the ports it uses, the environment variables n8n needs to run (some defined in the _.env_ file), and the volumes it needs to copy to the containers.
+
+Create a _docker-compose.yml_ file, make sure to create it in the same location as the _.env_ file:
+
+```shell
+nano docker-compose.yml
+```
+
+Add the following to the file:
+
+```yaml
+version: "3.7"
+
+services:
+  caddy:
+    image: caddy:latest
+    restart: unless-stopped
+    ports:
+      - "80:80"
+      - "443:443"
+    volumes:
+      - ${DATA_FOLDER}/caddy_data:/data
+      - ${DATA_FOLDER}/caddy_config:/config
+      - ${DATA_FOLDER}/caddy_config/Caddyfile:/etc/caddy/Caddyfile
+
+  n8n:
+    image: n8nio/n8n
+    restart: always
+    ports:
+      - 5678:5678
+    environment:
+      - N8N_BASIC_AUTH_ACTIVE=true
+      - N8N_BASIC_AUTH_USER
+      - N8N_BASIC_AUTH_PASSWORD
+      - N8N_HOST=${SUBDOMAIN}.${DOMAIN_NAME}
+      - N8N_PORT=5678
+      - N8N_PROTOCOL=https
+      - NODE_ENV=production
+      - WEBHOOK_URL=https://${SUBDOMAIN}.${DOMAIN_NAME}/
+      - GENERIC_TIMEZONE=${GENERIC_TIMEZONE}
+    volumes:
+      - ${DATA_FOLDER}/.n8n:/home/node/.n8n
+      - ${DATA_FOLDER}/local_files:/files
+
+volumes:
+  caddy_data:
+    external: true
+  caddy_config:
+```
+
+## Configure Caddy
+
+Caddy needs to know which domains it should serve, and which port to expose to the outside world. Create a _Caddyfile_ file in the *caddy_config* folder.
+
+```shell
+nano caddy_config/Caddyfile
+```
+
+Add the following configuration, adding your subdomain. The `n8n` tells Caddy to use the service definition defined in the _docker-compose.yml_ file:
+
+```text
+<SUB_DOMAIN> {
+    reverse_proxy n8n:5678 {
+      flush_interval -1
+    }
+}
+```
+
+## Start Docker Compose
+
+Start n8n and Caddy with the following command:
+
+```shell
+docker-compose up -d
+```
+
+Open the URL formed of the subdomain and domain name defined earlier, enter the user name and password defined earlier, and you should be able to access n8n.
+
+Stop n8n and Caddy with the following command:
```

Comment:
```suggestion
You can stop n8n and Caddy with the following command:
```

Reply:
I don't want to sound like we're ordering them to stop their instance. This makes it clearer it's an option I think.